post-processor/vagrant/libvirt.go

```diff
@@ -38,7 +38,9 @@ func (p *LibVirtProvider) Process(ui packer.Ui, artifact packer.Artifact, dir st
 	// Convert domain type to libvirt driver
 	var driver string
 	switch domainType {
-	case "kvm", "qemu":
+	case "none", "tcg":
```

Comment:
I'm not very familiar with the code, but are we sure "none" can be converted to qemu?